cdk_emqx_cluster/chaos_test.py

```diff
@@ -52,15 +58,55 @@ def __init__(self, scope: cdk.Construct, construct_id: str, cluster_name: str, *
                               )
 
                           }
-                          )
+                          ),
+            # * HIGH CPU LOAD 80%
+            SsmDocExperiment(self, id='emqx-high-cpu-80', name='AWSFIS-Run-CPU-Stress',
+                             desc='EMQX: CPU Stress 80% for 2mins',
+                             doc_parms={'DurationSeconds': '120', 'LoadPercent': '80'}),
+
+            # * HIGH CPU LOAD 100%
+            SsmDocExperiment(self, id='emqx-high-cpu-100', name='AWSFIS-Run-CPU-Stress',
+                             desc='EMQX: CPU Stress 100% for 2mins',
+                             doc_parms={'DurationSeconds': '120', 'LoadPercent': '100'}),
+
+            # * IO Stress
+            SsmDocExperiment(self, id='emqx-high-io-80', name='AWSFIS-Run-IO-Stress',
+                             desc='EMQX: IOStress 80% for 2mins',
+                             doc_parms={'DurationSeconds': '120', 'Percent': '80'}),
+
+            # * Kill emqx Process
+            SsmDocExperiment(self, id='emqx-kill-proc', name='AWSFIS-Run-Kill-Process',
+                             desc='EMQX: Kill emqx Process for 2mins',
+                             doc_parms={'ProcessName': 'beam.smp', 'Signal': 'SIGKILL'}),
+            # * Mem Stress
+            SsmDocExperiment(self, id='emqx-high-mem-80', name='AWSFIS-Run-Memory-Stress',
+                             desc='EMQX: Mem: 80% for 2mins',
+                             doc_parms={'DurationSeconds': '120', 'Percent': '80'}),
+
+            # * Network Blackhole
+            SsmDocExperiment(self, id='emqx-distport-blackhole', name='AWSFIS-Run-Network-Blackhole-Port',
+                             desc='EMQX: Drop distport inbond traffic for 2mins',
+                             doc_parms={'DurationSeconds': '120', 'Port': '4369', 'Protocol' : 'tcp', 'TrafficType':'ingress'}),
```

Comment:
Q: should we also affect port 4370?  Or is it only used by ekka and not by the Erlang Distribution?

Reply:
👍  right,  it should be port 4370